### PR TITLE
Action feedback for "View: Set horizontal zoom to default project set…

### PR DESCRIPTION
**Pull request by ScottChesworth**
…ting" (ID 41190)

### CODE REVIEW

**Comment by ScottChesworth** [comment]
@jcsteh, I feel like my branch names are getting longer and longer lol. What would you have called this?

**Review by jcsteh — CHANGES_REQUESTED** [review]
Thanks. Please update the readme as well.

**Comment by jcsteh** [comment]
Honestly, the branch name isn't super important when you're submitting it as a PR. I'm pretty lazy with them and tend to make it as short as possible without being too cryptic. I might have called this hZoomDefault or similar.

The commit message is another matter entirely. That should be as descriptive as necessary, as you have done here.

**Comment by ScottChesworth** [comment]
Oopsy, done now.

File: readme.md [hunk 1]
```diff
@@ -243,6 +243,7 @@ Most of these are actions built into REAPER, but a few are very useful actions f
 #### Zoom
 - Zoom out horizontal: - or NumPad-
 - Zoom in horizontal: = or NumPad+
+- Set horizontal zoom to default project setting
```
Comment (jcsteh) [inline readme.md:246]: All of these were missing the "View: " part of the action name, so I'm adding it to all of them here.
```suggestion
- View: Zoom out horizontal: - or NumPad-
- View: Zoom in horizontal: = or NumPad+
- View: Set horizontal zoom to default project setting
```

**Review by jcsteh — APPROVED** [review]

**Pull request merged**